extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.autoplay.js

```diff
@@ -2,23 +2,28 @@ define('wikia.articleVideo.featuredVideo.autoplay', [
 	'ext.wikia.adEngine.adContext',
 	'wikia.abTest',
 	'wikia.articleVideo.featuredVideo.cookies',
-], function (adContext, abTest, featuredVideoCookieService) {
-	var inFeaturedVideoClickToPlayABTest = abTest.inGroup('FV_CLICK_TO_PLAY', 'CLICK_TO_PLAY');
-
-	function isAutoplayEnabled() {
-		return featuredVideoCookieService.getAutoplay() !== '0' &&
-			// adContext.isEnabled('wgArticleVideoAutoplayCountries') &&
-			!adContext.get('rabbits.ctpDesktop') &&
-			!inFeaturedVideoClickToPlayABTest;
-	}
-
-	function inNextVideoAutoplayEnabled() {
-		// return adContext.isEnabled('wgArticleVideoNextVideoAutoplayCountries');
-		return true;
-	}
+	require.optional('ext.wikia.adEngine.ml.rabbit')
+], function (adContext, abTest, featuredVideoCookieService, rabbit) {
+	'use strict';
 
 	return {
-		isAutoplayEnabled: isAutoplayEnabled,
```

Comment:
Why? :) Wasn't it cleaner? (having returned interface short)

Reply:
It was cleaner but also impossible to test. For a method which takes into account X boolean variables I'd have to write X^2 test cases.